templates/support.rackspace.com/_includes/whitepaper-footer.html

```diff
@@ -0,0 +1,12 @@
+                        <p class="comment"><i class="fa fa-comments-o"></i> Continue the conversation elsewhere...</p>
+
+                        <aside class="cta">
+                            <h3>Start building today.</h3>
+                            <p class="button"><a href="https://cart.rackspace.com/cloud/">Sign Up Now</a></p>
+                            {#
```

Comment:
Yep! It's an artifact from self-service that I didn't remove when I did a copy-paste. I'll do that now.